### ISSUE TITLE
2.6.14748: TabbedProductionPanel#checkLists:152 - java.lang.IllegalStateException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#12615]
## Map
the_shogun / The Shogun

## Log Message
production_tabs: must include all player production rules/units; missing: buyFarmer, buyYumi-Sohei, buySohei, buyWako-Jong

## TripleA Version
[2.6.14748](https://github.com/triplea-game/triplea/releases/tag/2.6.14748)

## Java Version
11.0.19

## Operating System
Mac OS X

## Heap Size
2048M

## Stack Trace
```
Exception: java.lang.IllegalStateException production_tabs: must include all player production rules/units; missing: buyFarmer, buyYumi-Sohei, buySohei, buyWako-Jong
java.lang.Exception
	at games.strategy.triplea.ui.TabbedProductionPanel.checkLists(TabbedProductionPanel.java:152)
	at games.strategy.triplea.ui.TabbedProductionPanel.getRuleLists(TabbedProductionPanel.java:163)
	at games.strategy.triplea.ui.TabbedProductionPanel.initLayout(TabbedProductionPanel.java:62)
	at games.strategy.triplea.ui.ProductionPanel.show(ProductionPanel.java:103)
	at games.strategy.triplea.ui.TabbedProductionPanel.getProduction(TabbedProductionPanel.java:51)
	at games.strategy.triplea.ui.PurchasePanel$1.actionPerformed(PurchasePanel.java:71)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6635)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6400)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)


```

**Comment by TheDog-GH** [comment]
Im working on this.

**Comment by TheDog-GH** [comment]
Fixed with download version 1.4

@beelee1 Please close
